File: bmi.lua
io.write "Enter your weight in kg: "
weight = io.read()
io.write "Enter your height in meters: "
height = io.read()

bmi = weight / (height * height)

print("Your BMI is: "..bmi)

if bmi <= 18.5 then
    output="You're underweight."
elseif bmi <= 25 then
    output="You're healthy."
elseif bmi <= 30 then
    output="You're overweight."
else
    output="Holy McShizzle, you're freaking whale!"
end

print(output)

File: bmi.pl
print "Weight in kg: ";
$weight = <STDIN>;
print "Height in meters: ";
$height = <STDIN>;

$bmi = $weight / $height ** 2;
print "Your BMI is: ".$bmi."\n";

if($bmi <= 18.5)
{
    $output="You're underweight.";
}
elsif($bmi <= 25)
{
    $output="You're healthy.";
}
elsif($bmi <= 30)
{
    $output="You're overweight.";
}
else
{
    $output="Holy McShizzle, you're freaking whale!";
}

print $output . "\n";
